test/e2e/preset_test.go

```diff
@@ -608,6 +609,141 @@ func validateWorkspaceReadiness(workspaceObj *kaitov1alpha1.Workspace) {
 	})
 }
 
+// Create a temporary debug pod with curl if it doesn't already exist
+func createCurlDebugPod(namespace string) error {
+	By("Creating a temporary curl debug pod")
+
+	existingPod := &v1.Pod{}
+	err := utils.TestingCluster.KubeClient.Get(context.TODO(), client.ObjectKey{
+		Namespace: namespace,
+		Name:      curlPodName,
+	}, existingPod)
+
+	if err == nil {
+		By(fmt.Sprintf("Debug pod %s already exists", curlPodName))
+		return nil
+	} else if !errors.IsNotFound(err) {
+		return fmt.Errorf("failed to check existing pod: %v", err)
+	}
+
+	pod := &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      curlPodName,
+			Namespace: namespace,
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name:  "curl-container",
+					Image: "curlimages/curl",
+					Command: []string{
+						"sleep", "3600", // Keeps the pod running for long enough
```

Comment:
sleep "infinity"
this e2e usually last more than one hour 😂

Reply:
fixed